src/Rendering_Internals.md

```diff
@@ -0,0 +1,408 @@
+# Rendering Internals
+
+The Game Boy's PPU is the component responsible for feeding the LCD (= the screen) with pixels.
+This document describes how the PPU renders pixels.
+
+::: tip Terminology
```

Comment:
It appears that each instance of `tip` and `warning` have a leading space, thus breaking the parser and causing them to display incorrectly.

```suggestion
:::tip Terminology
```